src/components/shared/Navigation.tsx

```diff
@@ -1,16 +1,251 @@
-import { Flex, Text } from "@chakra-ui/react";
+/* eslint-disable react/no-multi-comp */
+import {
+  Flex,
+  Text,
+  Box,
+  Avatar,
+  Button,
+  Stack,
+  Menu,
+  MenuButton,
+  MenuList,
+  MenuItem,
+  Link,
+  Popover,
+  PopoverTrigger,
+  PopoverContent,
+  useColorModeValue,
+  VStack,
+} from "@chakra-ui/react";
+import Image from "next/image";
+import NextLink from "next/link";
+import Logo from "public/images/small_logo.svg";
 
 function Navigation() {
   // Assume that you know this information, nav bar changes based on this information
-  const isLoggedIn = false;
+  const isLoggedIn = true;
   const isChapter = false;
-  const isNonprofit = false;
+  const isNonprofit = true;
+  const npUserName = "liv.2b.girl";
+  const nonProftName = "Liv2BGirl";
+  const chapterName = "Chapter Name";
+
+  //colors
+  const black = "rgba(51, 51, 51, 1)";
```

Comment:
Instead of using rgba colors, just use the built in colors like "blue" or "blue.400".  The reason for that is that we will be adding a theme soon which will allow us to modify the colors throughout the entire app.